Marlin/pins_PRINTRBOARD_REVF.h

```diff
@@ -68,6 +71,28 @@
 
 #define BOARD_NAME         "Printrboard Rev F"
 #define LARGE_FLASH        true
+#ifndef USBCON
+  #error "USBCON should be defined by the platform for this board."
+#endif
+
```

Comment:
...and just checked BRAINWAVE (only 647 board and uses Bob's custom extension) and also identical 👍